app/Filament/Server/Pages/Startup.php

```diff
@@ -46,6 +47,7 @@ public function form(Form $form): Form
                         'lg' => 4,
                     ])
                     ->autosize()
+                    ->formatStateUsing(fn (Server $server, StartupCommandService $StartupCommandService) => $StartupCommandService->handle($server))
```

Comment:
_(suggestion below not pinted)_
```suggestion
                    ->formatStateUsing(fn (Server $server, StartupCommandService $service) => $service->handle($server, !auth()->user()->can(Permission::ACTION_STARTUP_READ, $server)))
```
https://github.com/pelican-dev/panel/blob/b355830db4b0a58e595d85ea82cc9d68511304c4/app/Transformers/Api/Client/ServerTransformer.php#L64

Reply:
I thought of this aswell but the page isn't even accessible without it so why double check ?
https://github.com/pelican-dev/panel/blob/b355830db4b0a58e595d85ea82cc9d68511304c4/app/Filament/Server/Pages/Startup.php#L156